roles/kube_lvm_storageclass/meta/main.yml

```diff
@@ -0,0 +1,2 @@
+dependencies:
```

Comment:
CI did it's job: you have to remove this now that the code is included in `kube_lvm_storageclass`

Reply:
This dependencies is required to set some variables used in the roles like bin_dir